utils/generate-bags/src/main.rs

```diff
@@ -48,30 +47,25 @@ impl Runtime {
 	}
 }
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, Parser)]
 struct Opt {
 	/// How many bags to generate.
-	#[structopt(long, default_value = "200")]
+	#[clap(long, default_value = "200")]
 	n_bags: usize,
 
 	/// Which runtime to generate.
-	#[structopt(
-		long,
-		case_insensitive = true,
-		default_value = "Polkadot",
-		possible_values = &Runtime::variants(),
-	)]
+	#[clap(long, ignore_case = true, arg_enum, default_value = "Polkadot")]
 	runtime: Runtime,
 
 	/// Where to write the output.
 	output: PathBuf,
 
 	/// The total issuance of the native currency.
-	#[structopt(short, long)]
+	#[clap(short, long)]
```

Comment:
I know that you not have done this, but do you know if that even makes sense? short and long?

Reply:
Yeah, it makes sense, `short` generates `-t` option, `long` generates `--total-issuance` option by default, or you could specify the name of options, like `short = 'a', long = "all"`